CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Add file linting using eslint version 18.1.0. https://github.com/sindresorhus/grunt-eslint.
 * Add build task to create a minified file and a concatenated file.
 * Move templates out of the javascript files and into their own html files.
+* Update readme with release instructions.
```

Comment:
i think it should. As @grahamgilchrist  said it should be here for pms etc to see.